starter_code/src/movies.js

```diff
@@ -1,19 +1,131 @@
 /* eslint no-restricted-globals: 'off' */
 
 // Iteration 1: All rates average - Get the average of all rates with 2 decimals 
+/*
+function ratesAverage(arr) {
+    const sum = arr.reduce((accumolator, currenValue) => {
+        if (currenValue.rate == "") {
+            return accumolator + 0
+        }
+        return accumolator + parseFloat(currenValue.rate)
+    }, 0) //<--- it starts with the accumolator
+    const avg = sum / arr.length;
+    return Math.round(avg * 100) / 100; //<--- Making two decimals after comma
+}
+*/
+
+function ratesAverage(arr) {
+    const ratesSum = arr.reduce((acc, val) => {
+        let rate = Number(val.rate);
+        if (!rate) rate = 0;
+        return acc + rate
+    }, 0)
+    const average = ratesSum / arr.length
+    return parseFloat(average.toFixed(2)); //<--- hmmm...
```

Comment:
`.toFixed(2)` transforms your number into a string with only the first two decimals
`parseFloat` on top will transform that string into a number